### ISSUE TITLE
word blacklisting

### ISSUE THREAD

**Issue by kyllian1212** [kyllian1212/Rin-rewrite#11]
as in add a word to a blacklist and the bot will spot if a message contains that word and report it/delete it

**Comment by edwardsew** [comment]
wondering if this is more suited to the new auto mod features at this point

**Comment by kyllian1212** [comment]
yeah i wonder too tbh, even dyno doing it is prob enough bc i like that u dont necessarily have to do 2 different searches in the log channel to check the discord automod messages and the manually reported messages
